### ISSUE TITLE
[Bug] Tab unit selection does not appear to be working on custom units

### ISSUE THREAD

**Issue by MutexMorgan** [KnuckleCracker/CW4-bug-tracker#385]
**Encountered on a custom colony sector map "Silos Aren't Just for Grain" by Alophox**
The bug is you can not tab select a custom unit, any other selection seem to work correctly.

The [screenshot](https://user-images.githubusercontent.com/26775462/101269005-b1980580-377a-11eb-99f5-e87163e66afa.jpg)
And the [output_log.txt](https://github.com/KnuckleCracker/CW4-bug-tracker/files/5648136/output_log.txt)

**Comment by sunyudai** [comment]
Not just custom units.

- is working for Emitters and Decals.
- is failing for: Everything else in the Unit Panel.

**Comment by Bob2Chiv** [comment]
related (possible duplicate of) #182

**Comment by MutexMorgan** [comment]
No, I said tab select unit, to build it while in-game, not in editor.
